exp/controllers/azuremachinepool_reconciler.go

```diff
@@ -87,3 +88,17 @@ func (s *azureMachinePoolService) Delete(ctx context.Context) error {
 
 	return nil
 }
+
+func (s *azureMachinePoolService) MachinePoolMachineScopeFromAmpm(ampm *infrav1exp.AzureMachinePoolMachine) *scope.MachinePoolMachineScope {
+	myscope, err := scope.NewMachinePoolMachineScope(scope.MachinePoolMachineScopeParams{
+		Client:                  s.scope.GetClient(),
+		MachinePool:             s.scope.MachinePool,
+		AzureMachinePool:        s.scope.AzureMachinePool,
+		AzureMachinePoolMachine: ampm,
+		ClusterScope:            s.scope.ClusterScoper,
+	})
+	if err != nil {
```

Comment:
We could probably just replace the whole return part with this since we're not doing anything special with the error.

```suggestion
return scope.NewMachinePoolMachineScope(scope.MachinePoolMachineScopeParams{
		Client:                  s.scope.GetClient(),
		MachinePool:             s.scope.MachinePool,
		AzureMachinePool:        s.scope.AzureMachinePool,
		AzureMachinePoolMachine: ampm,
		ClusterScope:            s.scope.ClusterScoper,
	})
```